packages/scoutgame/src/projects/getUserScoutProjects.ts

```diff
@@ -1,40 +1,40 @@
 import type { ScoutProject, ScoutProjectContract, ScoutProjectMemberRole } from '@charmverse/core/prisma-client';
 import { prisma } from '@charmverse/core/prisma-client';
 
-import { projectInfoSelect, projectSelect } from './projectSelect';
+import { projectMinimalSelect, projectDetailedSelect } from './projectSelect';
 
 export type ScoutProjectDetailed = Pick<
   ScoutProject,
   'id' | 'path' | 'avatar' | 'name' | 'description' | 'website' | 'github'
 > & {
-  contracts: Pick<ScoutProjectContract, 'id' | 'address' | 'chainId'>[];
+  contracts: Pick<ScoutProjectContract, 'id' | 'address' | 'chainId' | 'deployerId'>[];
   teamMembers: {
     id: string;
     path: string;
     avatar: string;
     displayName: string;
     role: ScoutProjectMemberRole;
   }[];
+  deployers: {
+    id: string;
+    address: string;
+  }[];
 };
 
-export type ScoutProjectMinimal = {
-  id: string;
-  path: string;
-  avatar: string;
-  name: string;
-};
+export type ScoutProjectMinimal = Pick<ScoutProject, 'id' | 'path' | 'avatar' | 'name'>;
 
 export async function getUserScoutProjects({ userId }: { userId: string }): Promise<ScoutProjectDetailed[]> {
```

Comment:
this method only needs to return minimal projects. it's used on your profile and to list projects - we don't need to show things like the deployer of contracts there